src/test/cypress/cypress/integration/Translation.spec.js

```diff
@@ -0,0 +1,338 @@
+                             firstEntryTitle, secondEntryTitle) {
+        cy.get("#opfab-navbar-menu-feed").should("have.text", feedTitle);
+        cy.get("#opfab-navbar-menu-archives").should("have.text", archivesTitle);
+        cy.get("#opfab-navbar-menu-monitoring").should("have.text", monitoringTitle);
+        cy.get("#opfab-navbar-menu-logging").should("have.text", loggingTitle);
+        cy.get("#opfab-navbar-menu-menu1").should("have.text", singleMenuTitle);
+        cy.get("#opfab-navbar-menu-dropdown-menu2").should("have.text", secondMenuTitle);
+        cy.get('#opfab-navbar-menu-dropdown-menu2').click();
+
+        // Test dropdown menus titles
+        cy.get('#opfab-navbar-menu-dropdown-menu2').click();
+        cy.get('.dropdown-menu').find('.text-link').eq(0).should("have.text", firstEntryTitle);
+        cy.get('.dropdown-menu').find('.text-link').eq(1).should("have.text", secondEntryTitle);
+    }
+
+    function checkRightMenuStaticEntries(realTimeTitle, settingsTitle, activityAreaTitle, notificationsTitle, aboutTitle,
+                                         changePasswordTitle, logoutTitle) {
+        
+        cy.get('#opfab-navbar-drop_user_menu').should("exist").click();
+        cy.get("#opfab-navbar-right-menu-realtimeusers").should("have.text", realTimeTitle);
+        cy.get("#opfab-navbar-right-menu-settings").should("have.text", settingsTitle);
+        cy.get("#opfab-navbar-right-menu-activityarea").should("have.text", activityAreaTitle);
+        cy.get("#opfab-navbar-right-menu-feedconfiguration").should("have.text", notificationsTitle);
+        cy.get("#opfab-navbar-right-menu-about").should("have.text", aboutTitle);        
+        cy.get("#opfab-navbar-right-menu-change-password").should("have.text", changePasswordTitle);
+        cy.get("#opfab-navbar-right-menu-logout").should("have.text", logoutTitle);
+    }
+
+    function checkDayAndNightTitles(dayModeTitle, nightModeTitle) {
+        
+        // Check day mode text
+        setNightMode();
+        cy.get('.opfab-right-menu').should('not.exist');
+        cy.get('#opfab-navbar-drop_user_menu').should("exist").click();
+        cy.get('.opfab-right-menu').should('exist');
+        cy.get("#opfab-navbar-right-menu-day-mode").should("have.text", dayModeTitle);
+        
+        // Check night mode text
+        setDayMode();
+        cy.get('.opfab-right-menu').should('not.exist');
+        cy.get('#opfab-navbar-drop_user_menu').should("exist").click();
+        cy.get('.opfab-right-menu').should('exist');
+        cy.get("#opfab-navbar-right-menu-night-mode").should("have.text", nightModeTitle);
+
+        // Close dropdown menu
+        cy.get('#opfab-navbar-drop_user_menu').should("exist").click();
+        cy.get('.opfab-right-menu').should('not.exist');
+    }
+
+
+
+    function checkArchiveScreenLabels(serviceLabel, tagLabel, processLabel, stateLabel, publishFromLabel, publishToLabel,
```

Comment:
the screen is called Archives so rename checkArchiveScreenLabels=>checkArchivesScreenLabels